rosbot_bringup/launch/bringup.launch.py

```diff
@@ -80,10 +91,34 @@ def generate_launch_description():
         executable="ekf_node",
         name="ekf_filter_node",
         output="screen",
-        parameters=[ekf_config],
+        parameters=[
+            ekf_config,
+            {
+                "map_frame": LaunchConfiguration(
+                    "ekf_map_frame", default=[namespace_tf_prefix, "map"]
+                )
+            },
```

Comment:
IMO, map should not be changed.
```suggestion
```

Reply:
When you use 2 robots the map frame would be in the place where the robot started mapping.
So every robot would have different start of mapping. To connect this 2 maps map it is needed an additional transform. For example: 
             ,- > robot1/map 
map - |
            `-> robot2/map